src/status_im2/contexts/quo_preview/tags/status_tags.cljs

```diff
@@ -25,32 +27,59 @@
     :options [{:value "Small"
                :key   :small}
               {:value "Large"
-               :key   :large}]}])
+               :key   :large}]}
+   {:label "Blur?"
+    :key   :blur?
+    :type  :boolean}])
 
 (defn cool-preview
   []
   (let [state (reagent/atom {:status :positive
-                             :size   :small})]
+                             :size   :small
+                             :blur?  false})]
     (fn []
-      (let [props (cond-> @state
-                    (= :positive (:status @state)) (assoc :status
-                                                          {:label (i18n/label :positive)
-                                                           :type  :positive})
-                    (= :negative (:status @state)) (assoc :status
-                                                          {:label (i18n/label :negative)
-                                                           :type  :negative})
-                    (= :pending (:status @state))  (assoc :status
-                                                          {:label (i18n/label :pending)
-                                                           :type  :pending}))]
+      (let [props (case (:status @state)
+                    :positive (-> @state
+                                  (assoc :status {:type :positive})
+                                  (assoc :label (i18n/label :t/positive)))
+                    :negative (-> @state
+                                  (assoc :status {:type :negative})
+                                  (assoc :label (i18n/label :t/negative)))
+                    :pending  (-> @state
+                                  (assoc :status {:type :pending})
+                                  (assoc :label (i18n/label :t/pending))))]
         [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
          [rn/view {:padding-bottom 150}
           [rn/view {:flex 1}
            [preview/customizer state descriptor]]
           [rn/view
-           {:padding-vertical 60
-            :flex-direction   :row
-            :justify-content  :center}
-           [quo2/status-tag props]]]]))))
+           {:style {:flex       1
+                    :margin-top 16}}
+           (when (:blur? @state)
+             [rn/view
+              {:style {:height        100
+                       :border-radius 16}}
+              [rn/image
+               {:source (resources/get-mock-image :community-cover)
+                :style  {:flex          1
+                         :width         "100%"
+                         :border-radius 16}}]
+              [blur/view
+               {:flex          1
+                :style         {:border-radius 16
+                                :height        100
+                                :position      :absolute
+                                :left          0
+                                :right         0}
+                :blur-amount   10
```

Comment:
this blur view with specific style is needed across many preview screens. Consider abstracting it out so we can use this easily on all preview screens. 
If it's too much for this pr I have a pr that needs it too and can also take a look at doing this 👍

Reply:
Sure, I will abstract this and place it under the  `quo_preview > common` directory.